packages/ckeditor5-table/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
-  "extends": "ckeditor5/tsconfig.json",
+  "extends": "../../tsconfig.json",
```

Comment:
Why such change?

Reply:
To make it consistent with `tsconfig.release.json` in the same package.